service/src/main/kotlin/fi/espoo/evaka/daycare/controllers/DaycareController.kt

```diff
@@ -619,6 +619,12 @@ class DaycareController(
             .also { Audit.UnitView.log(targetId = AuditId(unitId)) }
     }
 
+    private fun getMissingGroupPlacementsWithinSixMonths(
+        tx: Database.Read,
+        unitId: DaycareId,
+        today: LocalDate,
+    ) = getMissingGroupPlacements(tx, unitId).filter { it.gap.start <= today.plusMonths(6) }
```

Comment:
`getMissingGroupPlacements` is only used in these two places. How about including the 6 month limit to the SQL instead?

Reply:
Moved the filtering now to the query function. I suppose it could also go within the SQL itself, but I'm a bit unsure how to do that since it's based on `unnest(p.ranges) AS gap`. I could of course wrap the result in another CTE and add a WHERE-clause to that, but the readability downside might not be worth the performance upside.